src/NetworkTestWidget/Tests/Tests.ts

```diff
@@ -1,9 +1,17 @@
 import { testBitrate, BitrateTest } from '@twilio/rtc-diagnostics';
 import { Device, Connection, PreflightTest } from 'twilio-client';
+import { getLogger } from 'loglevel';
 import { regionalizeIceUrls } from '../../utils';
 import { Region } from '../../types';
 import RTCSample from 'twilio-client/es5/twilio/rtc/sample';
 
+const log = getLogger(require('../../../package.json').name);
```

Comment:
This will include all package.json content in the js artifacts, which includes private information added by npm such as local directory path. We should import this at the top instead.
```ts
import { name as appName } from '../../../../package.json';

const log = getLogger(appName);
```

Reply:
Ah I remember this issue. Nice catch!